dotnet/dotnet-cloud-run-hello-world/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine AS base
+WORKDIR /src
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine AS build
+WORKDIR /src
+COPY . .
+RUN dotnet restore dotnet-cloud-run-hello-world.csproj
+RUN dotnet build "./dotnet-cloud-run-hello-world.csproj" -c Debug -o /out
+
+FROM build AS publish
+RUN dotnet publish dotnet-cloud-run-hello-world.csproj -c Debug -o /out
+
+# Building final image used in running container
+FROM base AS final
```

Comment:
The distinct declaration of base which is not used otherwise seems unnecessary. If we're going to include it as a matter of training developers on multi-stage caching strategies, we should include a comment explaining it.

Should we add this to Java & Go?

Reply:
Yes, I think we should add it to other languages too and good point on the comment. I'll add it.